source/features/pagination-hotkey.tsx

```diff
@@ -21,19 +22,34 @@ const previousPageButtonSelectors = [
 function init(): void {
 	const createNextPageButton = select(nextPageButtonSelectors);
 	if (createNextPageButton) {
-		createNextPageButton.dataset.hotkey = 'ArrowRight';
+		createNextPageButton.dataset.hotkey = pageDetect.isRepoConversationList() ? 'ArrowRight' : 'n';
 	}
 
 	const createPreviousPageButton = select(previousPageButtonSelectors);
 	if (createPreviousPageButton) {
-		createPreviousPageButton.dataset.hotkey = 'ArrowLeft';
+		createPreviousPageButton.dataset.hotkey = pageDetect.isRepoConversationList() ? 'ArrowLeft' : 'p';
 	}
 }
 
 void features.add(__filebasename, {
-	shortcuts: {
-		'→': 'Go to the next page',
-		'←': 'Go to the previous page',
-	},
+	shortcuts: (pageDetect.isRepoConversationList())
+		? {
+			'←': 'Go to the previous page',
+			'→': 'Go to the next page',
+		}
+		: {
+			n: 'Go to the next page',
+			p: 'Go to the previous page',
+		},
+	include: [
+		pageDetect.isConversationList,
+		pageDetect.isLabelList,
+		pageDetect.isNotifications,
+		pageDetect.isRepoCommitList,
+	],
```

Comment:
`isDiscussionsList` also has pagination, but the elements there are weird, there are multiple `.pagination` elements that is breaking the existing selectors ( though ideally, it should have worked ).

Reply:
Let's add a `// TODO:` comment with this information, shortened

Reply:
I could test and add the extra selectors; but then it would no longer be a **Meta** change though. 
Should I add the extra selectors to this pull request? Or should I create a separate pull request after this is merged in?

Reply:
You can make all the changes here

Reply:
Yes, that is exactly [how I feel](https://github.com/sindresorhus/refined-github/pull/4743#discussion_r698768658) too. Getting to work on it.